packages/select/test/multiSelectTests.tsx

```diff
@@ -11,18 +11,16 @@ import * as React from "react";
 import * as sinon from "sinon";
 
 // this is an awkward import across the monorepo, but we'd rather not introduce a cyclical dependency or create another package
```

Comment:
in a future PR, we should move this to `packages/select/src/__fixtures__/`. then you'd do the import `from "@blueprintjs/select/src/__fixtures__/films"`

btw we'd want to `.npmignore` `__tests__` and `__fixtures__`

Reply:
cool yes i like that